torch/_inductor/graph.py

```diff
@@ -90,6 +90,8 @@ def supported_dtype_of_cpp_wrapper(dtype, cuda):
     if cuda:
         supported_dtype.add(torch.float8_e4m3fn)
         supported_dtype.add(torch.float8_e5m2)
+        supported_dtype.add(torch.float8_e4m3fnuz)
+        supported_dtype.add(torch.float8_e5m2fnuz)
```

Comment:
Should we add those only if `hip`? Or `cuda` is alias for `hip`?

Reply:
What is the danger in adding them unconditionally?  fnuz is not just a hip thing.